embetcd/client.go

```diff
@@ -84,30 +84,57 @@ func (c *Client) PutWithKeepAlive(ctx context.Context, key string, value string,
 	return lease, keepAlive, cancel, err
 }
 
+func (c *Client) arePeerURLSInCluster(ctx context.Context, apURLS []url.URL) (areIn bool, err error) {
+	var members *cli.MemberListResponse
+	members, err = c.MemberList(ctx)
+
+	if err == nil && members != nil && members.Members != nil {
+		apStrings := URLSToStringSlice(apURLS)
+
+		for _, member := range members.Members {
+			memberPURLs := member.GetPeerURLs()
+			for _, u := range apStrings {
+				if StringIsInStringSlice(u, memberPURLs) {
+					areIn = true
+					break
```

Comment:
This is only checking if one of the apURLS is in the MemberList, instead of all the apURLS. Is that the expected behavior?

Reply:
@sstewart-signalfx  yep, if any of the members have a conflicting peer url we the answer is true.